asllib/Interpreter.ml

```diff
@@ -858,7 +859,9 @@ module Make (B : Backend.S) (C : Config) = struct
        | _ -> Format.eprintf "@[<3>Eval@ @[%a@]@]@." PP.pp_stmt s);
     match s.desc with
     (* Begin SPass *)
-    | S_Pass -> return_continue env |: SemanticsRule.SPass
+    | S_Pass ->
+       let*| () = B.noAction () in
```

Comment:
You are right, this change is not useful.

I was afraid of empty else clauses. In that case we would need the `pass` statement to generate some effect, in order to have a target for the `iico_control` dependency. But in face we do not need that, because else clauses of conditional branches are not empty: the procedure `BranchNotTaken` is called , and the default procedure is overridden by [ours](https://github.com/herd/herdtools7/blob/d9f232862fc374c166f4404afd7f98c6cde51bbf/herd/libdir/asl-pseudocode/patches.asl#L122) that updates the `_PC` register and hence generate effects.

I'll cancel this change. Thanks.